Initial commit of helm-charts for Tuxedo v22 (#12)
oracle/helm-charts: tuxedo/charts/tuxedo-ws/README.md

> # Oracle Tuxedo WS (workstation) SSL server

Reviewer: ```suggestion
# Oracle Tuxedo Workstation SSL server
```

> This helm chart creates the Tuxedo WS (workstation) SSL server with two services: `BASICWS` and `TOUPPER`

Reviewer: ```suggestion
This Helm chart creates an Oracle Tuxedo Workstation (WS) SSL server with two services: `BASICWS` and `TOUPPER`
```

## Prerequisites

>   1. Build the `tuxedows_svr` container image using the [Oracle Tuxedo WS App](https://github.com/oracle/docker-images/OracleTuxedo/core/samples/ws_ssl_svr/).

Reviewer: ```suggestion
  1. Build a `tuxedows_svr` container image using the [sample Oracle Tuxedo WS App](https://github.com/oracle/docker-images/OracleTuxedo/core/samples/ws_ssl_svr/).
```

>   2. Upload the container image to a private container registry. This location will be used in the `helm install` step later in this procedure.

Reviewer: ```suggestion
  2. Upload the `tuxedows_svr` image you built in step 1 to a private container registry and take a note of its location. This location will be used in the `helm install` step later in this procedure.
```

  3. If your client will be connecting over an external IP connection, install the `loadbal` helm chart to obtain the external IP address as follows:
     ```shell
     EXT_IP=$(kubectl get service --namespace default tuxedo-loadbal-svc --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
     ```

>      This IP will be used to access the tuxedo services on port 9055, 9060 and 2071 to 2075 from outside the kubernetes cluster.

Reviewer: ```suggestion
  3. If you want to be able to connect to the Tuxedo WS server from an external source, use the `loadbal` helm chart to obtain an external IP address:
  
     ```shell
     EXT_IP=$(kubectl get service --namespace default tuxedo-loadbal-svc --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
     ```

     This IP address will provide external access to the Tuxedo services running on your Kubernetes cluster on ports 9055, 9060 and 2071 to 2075.
```

## Create the Tuxedo WS SSL server

> To create the `Tuxedo WS SSL server` using this Helm chart, run the following command:

Reviewer: ```suggestion
To create a Tuxedo WS SSL server using this Helm chart, run:
```
